README.md

```diff
@@ -50,25 +66,44 @@ if resp.status == 200:
 else:
     sys.exit(1)
 ```
-Add execution permissions to it: `chmod +x /tmp/healthcheck.py`.
 
-#### 4. Build horust:
+Don't forget to make it executable:
+
+```shell
+chmod +x /tmp/healthcheck.py`.
+```
+
+#### 4. Build Horust:
+
 This step is only required because we don't have a release yet, or if you like to live on the edge.
-For building horust, you will need [rust](https://www.rust-lang.org/learn/get-started). 
-As soon as it's installed, you can build Horust with cargo using: `cargo build --release`
 
-#### 4. Run horust: "./horust". By default it will search services inside the `/etc/horust/services` folder.
-Now every 10 seconds, this will send an http head request to google.it. 
-If the response is different than 200, then there is an issue!
+For building Horust, you will need [Rust](https://www.rust-lang.org/learn/get-started). As soon as it's installed, you can build Horust with Rust's `cargo`: 
+
+```shell
+cargo build --release
+```
 
-In this case, we're just exiting with a different exit code. But in real life, you could trigger other actions.
-Use ctrl+c to stop Horust. Horust will send a SIGTERM signal to all the running services, and if it doesn't hear back for a while, it will terminate them via a SIGKILL.
+#### 4. Run Horust:
 
----
+Now you can just:
+
+```shell
+./horust
+```
+
+By default Horust searches for services inside the `/etc/horust/services` folder (which we have created in step 1).
 
-Check the [documentation](https://github.com/FedericoPonzi/Horust/blob/master/DOCUMENTATION.md) for a complete reference of the options available on the service config file.
+Every 10 seconds from now on, Horust will send an HTTP `HEAD` request to https://google.it. If the response is different than `200`, then there is an issue!
 
-> You can also bootstrap the creation of a new service, by using `horust --sample-service > new_service.toml`.
+In this case, we're just exiting with a different exit code (i.e. a `1` instead of a `0`). But in real life, you could trigger other actions - maybe storing this information in a database for long-term analysis, or sending an e-mail to the website's owner.
+
+#### 5. Finish up:
+
+Use <kbd>Ctrl</kbd>+<kbd>C</kbd> to stop Horust. Horust will send a `SIGTERM` signal to all the running services, and if it doesn't hear back for a while - it will terminate them by sending an additional  `SIGKILL` signal.
```

Comment:
I wasn't aware of this kdb tag, pretty cool!

Reply:
That it is:)